catalog/transformer/catalog-transformer-csv/src/main/java/ddf/catalog/transformer/csv/MetacardIterator.java

```diff
@@ -22,13 +22,16 @@
 import java.util.Iterator;
 import java.util.List;
 import java.util.NoSuchElementException;
+import org.apache.commons.lang3.StringUtils;
 
 /**
  * An implementation of java.util.Iterator which iterates over Metacard attribute values.
  *
  * @see java.util.Iterator
  */
 class MetacardIterator implements Iterator<Serializable> {
+  private static final String MULTIVALUE_DELIMITER = "\n";
```

Comment:
❓ is a newline valid for a single csv row or is it escaped later down the line?

Reply:
A newline for an attribute value?

Reply:
If this is being used to create a CSV, then wouldn't a  newline be a reserved character used for delimiting a new row within the CSV? It seems like multivalues would mess up that row. 
meaning:
valid csv:
```csv
singlevalueattr, multivalueattr
value1, multivalue1a|multivalue1b
value2, multivalue2a|multivalue2b
```

but if you use a `\n` as opposed to (abitrary delimiter for sake of example) `|` 
then you would mess up the CSV if that makes sense. It would result in CSV looking like this
```csv
singlevalueattr, multivalueattr
value1, multivalue1a
multivalue1b
value2, multivalue2a
multivalue2b
```

Unless of course i'm missing something and the newline literals aren't actually being added into the end CSV (but looking at the code it looks like it uses the iterator to print the CSV)

Reply:
Hmm, yeah, looking at the expected value (on this test)[https://github.com/codice/ddf/pull/2821/files#diff-e0a03d03376fd714af5d936ad29d6a8aR91], I think that is a problem

Reply:
Newlines are special characters but the `CsvQueryResponseTransformer` uses a `CSVPrinter` that is smart enough to surround values that contain newline chars with quotes, which allows Excel to honor the multiline values as a single field. It's not the prettiest output though so I'm still open to changing the delimiter char to `|`. Thoughts?
<img width="650" alt="screen shot 2018-01-09 at 8 51 37 am" src="https://user-images.githubusercontent.com/8922441/34729728-bfe393e8-f51a-11e7-8f38-797c082acaaa.png">
Idk if Excel is just smart enough to

Reply:
Cool. As long as it works you can leave it however y'all wish. Just double checking